config/webpack.prod.js

```diff
@@ -21,7 +21,7 @@ const webpackMerge = require('webpack-merge');
  * Webpack Constants
  */
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
-const API_URL = process.env.API_URL || 'http://api.almighty.io/api/';
+const API_URL = process.env.API_URL || 'http://api.prod-preview.openshift.io/api/';
```

Comment:
I don't understand because in DEV we use localhost URL. 
So do we introduce a new ENV variable? Why would you give WIT (work item) in the variable name?